### PR TITLE
metrics: add asynchronous write io panels

### LINKED ISSUE #10540: raftstore: support asynchronous write io
## Development Task
The sync write io in store thread introduces much extra latency. After detailed analysis, we found that it increases commit duration by about one IO+fsync time. After tikv/raft-rs#403, we can use async write io in store thread.

## Task list(updated 2021/9/8)

- [x] Support asynchronous write io in raftstore https://github.com/tikv/tikv/pull/10289
It is split into several PRs for review easily. 
https://github.com/tikv/tikv/pull/10326
https://github.com/tikv/tikv/pull/10667
https://github.com/tikv/tikv/pull/10672
    - [x] Dedicated io threads 
    - [x] Change snapshot process
    - [x] Change peer destroy process
    - [x] Unpersisted entries must exist in entry cache
    - [x] Support hotpot scheduling
    - [x] Simple flow control
    - [x] Waterfall metrics
    - [x] Intergration Tests

### PR DESCRIPTION
**Pull request by gengliqi**
### What problem does this PR solve?

Issue Number: close #10540

Problem Summary:

### What is changed and how it works?

What's Changed:
Add raft waterfall some other related panels to grafana.
image

### Related changes

- Need to cherry-pick to the release branch

### Check List 

Tests 

- No code

Side effects

- No

### Release note 

```release-note
None.
```

### CODE REVIEW

File: metrics/grafana/tikv_details.json [hunk 1]
```diff
@@ -34717,7 +37506,7 @@
       "type": "row"
     }
   ],
-  "refresh": "1m",
+  "refresh": "30s",
```
Comment (BusyJay) [inline metrics/grafana/tikv_details.json:37509]: Why change this?

File: metrics/grafana/tikv_details.json [hunk 2]
```diff
@@ -26244,6 +29026,13 @@
           },
           "links": [],
           "reverseYBuckets": false,
+          "scopedVars": {
```
Comment (BusyJay) [inline metrics/grafana/tikv_details.json:29029]: What is it used for?

File: metrics/grafana/tikv_details.json [hunk 3]
```diff
@@ -15708,7 +18740,7 @@
               "step": 10
             },
             {
-              "expr": "sum(rate(tikv_scheduler_latch_wait_duration_seconds_sum{tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\", type=\"$command\"}[1m])) / sum(rate(tikv_scheduler_latch_wait_duration_seconds_count{tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\", type=\"$command\"}[1m])) ",
```
Comment (BusyJay) [inline metrics/grafana/tikv_details.json:15711]: Why change this?

**Review by BusyJay — COMMENTED** [review]
There seems to be a lot of unnecessary changes.

**Review by BusyJay — APPROVED** [review]

**Review by innerr — APPROVED** [review]
LGTM

**Pull request merged**